mantle/kola/tests/ignition/qemufailure.go

```diff
@@ -37,6 +40,14 @@ func init() {
 		Platforms:   []string{"qemu"},
 		Tags:        []string{"ignition"},
 	})
+	register.RegisterTest(&register.Test{
+		Name:        "coreos.ignition.uniquebootfs",
+		Description: "Verify there is only one bootfs.",
```

Comment:
So it looks like you're working on the first of the two tests suggested in https://github.com/coreos/coreos-assembler/issues/2953. That test isn't actually related to Ignition, so I'd name it e.g. `coreos.unique.boot.failure`. The `failure` bit is to allude to the fact that we're testing that we correctly fail given certain conditions and to be consistent with the naming of the other test in this file.

For the description, maybe:

> Verify boot fails if there are multiple boot filesystems.

?